packages/slice-machine/lib/models/common/Slice.ts

```diff
@@ -21,6 +21,17 @@ const Slice = {
   },
 };
 
+export interface SliceCreateBody {
```

Comment:
OPT: you know you could have a base type and extend it?
```
interface SliceBody {
  sliceName: string;
  from: string;
}

interface SliceCreateBody extends SliceBody {
   values?: { componentCode: string; model: Models.SliceAsObject };
}
```

Reply:
Done